engine/consensus/matching/engine.go

```diff
@@ -80,9 +82,10 @@ func New(
 	e := &Engine{
 		unit:                    engine.NewUnit(),
 		log:                     log.With().Str("engine", "matching").Logger(),
-		metrics:                 collector,
+		collector:               collector,
```

Comment:
feel like if we're going to rename this, we could consider making it more descriptive in naming.